doc/content/toolstack/features/NUMA/index.md

```diff
@@ -139,17 +159,17 @@ Meaning of the policy:
 * `best_effort`: the algorithm described in this document, where soft pinning is used to achieve better balancing and lower latency
 * `default_policy`: when the admin hasn't expressed a preference
 
-* Currently `default_policy` is treated as `any`, but the admin can change it, and then the system will remember that change across upgrades.
+* Currently, `default_policy` is treated as `any`, but the admin can change it, and then the system will remember that change across upgrades.
    If we didn't have a `default_policy` then changing the "default" policy on an upgrade would be tricky: we either risk overriding an explicit choice of the admin, or existing installs cannot take advantage of the improved performance from `best_effort`
-* Future XAPI versions may change `default_policy` to mean `best_effort`.
+* Future, XAPI versions may change `default_policy` to mean `best_effort`.
```

Comment:
The comma shouldn't be there, "Future XAPI versions" is the proper grammatical subject here
```suggestion
* Future XAPI versions may change `default_policy` to mean `best_effort`.
```